airflow/providers/openlineage/utils/sql.py

```diff
@@ -33,6 +34,9 @@
     from airflow.hooks.base import BaseHook
 
 
+log = logging.getLogger(__name__)
```

Comment:
Same here - it seems that while helpful to debug, likely we could have a separate them out - they add a bit noise to the change.

Reply:
Same as ⬆️